nntrainer/tensor/blas_interface.h

```diff
@@ -162,20 +162,36 @@ void ele_mul(const unsigned int N, const _FP16 *X, const _FP16 *Y, _FP16 *Z,
              unsigned int o_stride = 1);
 
 /**
- * @brief     elementwise vector addition : Z = X + alpha * Y + beta *
+ * @brief     elementwise vector addition : Z = X + beta * Y + gamma *
  * Z
  * @param[in] N  length of the vector
  * @param[in] X _FP16 * for Vector X
  * @param[in] Y _FP16 * for Vector Y
  * @param[in] Z _FP16 * for Vector Z
- * @param[in] alpha scalar multiplier for input
- * @param[in] beta scalar multiplier for output
+ * @param[in] beta scalar multiplier for input
+ * @param[in] gamma scalar multiplier for output
  * @param[in] i_stride input stride
  * @param[in] o_stride output stride
```

Comment:
let's clarify whether the input is X or Y.

```suggestion
 * @param[in] beta scalar multiplier for input Y
 * @param[in] gamma scalar multiplier for output
 * @param[in] i_stride input Y stride
 * @param[in] o_stride output stride
```